NuRadioReco/detector/generic_detector.py

```diff
@@ -28,8 +29,9 @@ class GenericDetector(NuRadioReco.detector.detector.Detector):
     This detector only accepts json detector descriptions or dictionary.
     """
 
-    def __init__(self, json_filename, default_station, default_channel=None, default_device = None, source='json', dictionary=None,
-                 assume_inf=True, antenna_by_depth=True):
+    def __init__(self, json_filename, default_station=None, default_channel=None, default_device=None, source='json',
+                 dictionary=None,
+                 assume_inf=True, antenna_by_depth=False):
```

Comment:
I see that you changed the default of `antenna_by_depth`. Great. I was just about to suggest that ;-)

Reply:
Thanks go to Lilly